addons/sale_timesheet/tests/test_project_update.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+
+from datetime import date
+from dateutil.relativedelta import relativedelta
+
+from odoo.addons.project.tests.test_project_update import TestProjectUpdate
+from odoo.tests import tagged
+from odoo.tests.common import Form
```

Comment:
Based on the [guidelines](https://www.odoo.com/documentation/master/reference/guidelines.html#imports), we should do it like this:
```suggestion
from datetime import date
from dateutil.relativedelta import relativedelta

from odoo.tests import tagged
from odoo.tests.common import Form

from odoo.addons.project.tests.test_project_update import TestProjectUpdate
```